CONTRIBUTING.md

```diff
@@ -28,5 +28,25 @@ We use GitHub issues to track public bugs. Please ensure your description is cle
 * "Attractive"
 * Do not use the optional parameters of `setTimeout` and `setInterval`
 
+## Testing
+
+To run tests in the browser, run
+
+```
+npm run test:server
+```
+
+Open http://localhost:8081/test. Anytime you change test files, the tests will automatically be rerun, no refresh
+required! You can even click a specific section to only run those tests.
+
+To run tests in the CLI, run
+
+```
+npm run test
+```
+
+This will fail on some errors that npm run test:server won't, so it's recommended that you run this command after you've
+finished developing on tests with npm run test:server, if you chose that method.
```

Comment:
Here are the script differences, so you can see what I'm talking about. package.json scripts:
```
    "test": "mocha-webpack --webpack-config webpack.config-test.js \"test/**/*-test.js\" --require build_helpers/test-globals.js",
    "test:watch": "mocha-webpack --webpack-config webpack.config-test.js --watch \"test/**/*-test.js\" --require build_helpers/test-globals.js",
    "test:server": "webpack-dev-server --config webpack.config-test.js --hot --inline"
```